Testscripts/Linux/SetupRDMA.sh

```diff
@@ -86,6 +86,17 @@ function Main() {
 		echo 8 > /etc/yum/vars/releasever
 		LogMsg "$?: Applied a mitigation for $DISTRO to /etc/yum/vars/releasever"
 	fi
+	# CentOS-HPC 7.6 or older versions only support ND device.
```

Comment:
CentOS 7.6 HPC is for SR-IOV only

Reply:
@johnsongeorge-w , can you check out LIS code? It looks contradictory.

Reply:
@juhlee-microsoft This is correct. RDMA over ND LIS is till CentOS7.5

Reply:
@johnsongeorge-w  what does LIS code show then? This is the exact question I asked and I received this line from you. # Network direct driver in LIS is supported for RHEL/CentOS <= 7.6 
Does it mean LIS code in incorrect?

Reply:
fixed to 7.5 or earlier.

Reply:
ba42dda386242275442db94f8ee820f4757ac75d